mobile/src/reducers/organization/organization.epics.ts

```diff
@@ -1,40 +1,49 @@
 import { ActionsObservable } from 'redux-observable';
-import { LoadDepartments, loadDepartmentsFinished, LoadDepartmentsFinished, loadEmployeeFinished, LoadEmployeesForDepartment, loadEmployeesForDepartment } from './organization.action';
+import { LoadDepartments, loadDepartmentsFinished, LoadDepartmentsFinished, loadEmployeeFinished, LoadEmployeesForDepartment, loadEmployeesForDepartment, ErrorLoadFailed } from './organization.action';
 import { deserializeArray, deserialize } from 'santee-dcts/src/deserializer';
 import { Department } from './department.model';
 import { ajaxGetJSON } from 'rxjs/observable/dom/AjaxObservable';
 import { AppState } from '../app.reducer';
 import { Employee } from './employee.model';
+import { Observable } from 'rxjs/Observable';
+import { Alert } from 'react-native';
 
 const url = 'http://localhost:5000/api'; //TODO: fix hardcode
+export const errorEpic$ = (action$: ActionsObservable<ErrorLoadFailed>) =>
+    action$.ofType('ERROR-LOAD-FAILED')
+        .do(({ errorMessage }) => Alert.alert('Error', `error occurred: ${errorMessage}`))
+        .ignoreElements();
 
 export const loadDepartmentsEpic$ = (action$: ActionsObservable<LoadDepartments>) =>
     action$.ofType('LOAD-DEPARTMENTS')
         .switchMap(x => ajaxGetJSON(`${url}/departments`))
         .map(x => deserializeArray(x as any, Department))
-        .map(x => loadDepartmentsFinished(x));
+        .map(x => loadDepartmentsFinished(x))
+        .catch(e => Observable.of({ type: 'ERROR-LOAD-FAILED', errorMessage: e.message || 'something wrong' }));
```

Comment:
you have a function that creates these instances, why do you create them manually? -> `errorLoadFailed(e.message)`